submissions/kris-almort/html-css-popup/index.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" href="css/main.css">
```

Comment:
Your app doesn't display correctly on phones. You can try it yourself on your smartphone. I think, it is because, in spite of you've added media queries, you didn't add `<meta name="viewport" content="width=device-width, initial-scale=1">` in the `head` of your html file. Try to play with it.

Reply:
You were right. Now I tested on mobile, fixed a bug, plus fixed an additional bug with the display of the input on a mobile safari. Done in commit 6f2bd3b

Reply:
Much better 🐱